envoyauth/evaluation.go

```diff
@@ -32,8 +31,7 @@ type EvalContext interface {
 
 // PrepareQueryOpts - Options to prepare a Rego query to be passed to the CreatePreparedQueryOnce method
 type PrepareQueryOpts struct {
-	Opts        []func(*rego.Rego)
-	PrepareOpts []rego.PrepareOption
+	Opts []func(*rego.Rego)
```

Comment:
This would be a breaking change. I'm not quite sure why we need to remove this and what the purpose of the change is. It would be helpful to understand this. Thanks!

Reply:
There is no usage of this field. However, I could revert this change if it has concerns.

Reply:
Let's revert the change in this PR as it's unrelated and also backwards-incompatible. Thanks.

Reply:
PTAL